lms/templates/ccx/enrollment.html

```diff
@@ -4,15 +4,16 @@
   <form method="POST" action="ccx_invite">
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
   <h2> ${_("Batch Enrollment")} </h2>
-  <p>
-    <label for="student-ids">
+  <label for="student-ids" class="sr">${_("Email Addresses/Usernames")}</label>
+  <p id="label_student_ids">
```

Comment:
I think this tag should still come before the label unless there is something I'm missing.